{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--50172803-838e-4e6c-8681-c28e6f45702e",
    "created": "2020-05-26T20:30:04.747556Z",
    "modified": "2020-05-26T20:30:04.747556Z",
    "name": "CVE-2019-20426",
    "description": "In the Lustre file system before 2.12.3, the ptlrpc module has an out-of-bounds access and panic due to the lack of validation for specific fields of packets sent by a client. In the function ldlm_cancel_hpreq_check, there is no lock_count bounds check."
}